### PR TITLE
Sync data between PR list and menu bar PR list

### PR DESCRIPTION
**Pull request by Ckitakishi**
## Description
Sync my PR list data between PR list and menu bar PR list

### CODE REVIEW

File: PRChecker/Shared/Unique Views/PullRequestCell.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ import SwiftUI
 
 struct PullRequestCell: View {
     
-    @StateObject var pullRequest: PullRequest
+    let pullRequest: PullRequest
```
Comment (brevansio) [inline PRChecker/Shared/Unique Views/PullRequestCell.swift:12]: Maybe this should be ObseveredObject so it refreshes?

**Review by brevansio — COMMENTED** [review]
Looks good but I'm stuck wondering about switching to ObservedObject x4 instead of just let.

File: PRChecker/Shared/Unique Views/PullRequestCell.swift (hunk 1 above)
Comment (Ckitakishi) [reply PRChecker/Shared/Unique Views/PullRequestCell.swift:12]: I think ObservedObject x4 also works, but I need to convert `PullRequest.Header/Content/Footer` from structure to class first for adopting `ObservableObject` protocol

Comment (brevansio) [reply PRChecker/Shared/Unique Views/PullRequestCell.swift:12]: Ah that's true. We can probably skip that but I think it's worth it to do the full PR at least.

Comment (Ckitakishi) [reply PRChecker/Shared/Unique Views/PullRequestCell.swift:12]: I agree!

**Pull request merged**